packages/unpublished-model-encapsulation-test-app/ember-cli-build.js

```diff
@@ -5,8 +5,20 @@
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function(defaults) {
+  const isProd = process.env.EMBER_ENV === 'production';
+  const compatWith = process.env.EMBER_DATA_FULL_COMPAT ? '99.0' : null;
```

Comment:
Whats this about? Can you please add a comment

Reply:
this ensures deprecation stripping when desired as well, matching the config and flag used by the tests in -ember-data package